src/main/java/io/stargate/sgv2/jsonapi/exception/DatabaseException.java

```diff
@@ -9,8 +9,14 @@ public DatabaseException(ErrorInstance errorInstance) {
   }
 
   public enum Code implements ErrorCode<DatabaseException> {
+    CLOSED_CONNECTION,
```

Comment:
hold on merging this - we need to work out how to hide errors like the ones below from the users.

- CLOSED_CONNECTION,
- DRIVER_TIMEOUT,
    NO_NODE_AVAILABLE,

also, it is sloppy to have a TABLE_WRITE_TIMEOUT, and WRITE_TIMEOUT; we need to work out if these are table specific errors or not.